docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
     environment:
       - discovery.type=single-node
       - cluster.name=workflow.elasticsearch
+      - cluster.routing.allocation.disk.threshold_enabled=false
```

Comment:
On running `initializeEs` I am encountering the error `TOO_MANY_REQUESTS/12/ disk usage exceeded flood-stage watermark, index has read-only-allow-delete block`, preventing index creation

This solution unblocks me & comes from https://stackoverflow.com/a/75962819